qrisk/stats.py

```diff
@@ -15,13 +15,10 @@
 
 from __future__ import division
 
-from collections import OrderedDict
-from functools import partial
-
 import pandas as pd
 import numpy as np
 import scipy as sp
-import scipy.stats
+import scipy.stats  # noqa
```

Comment:
I can't explain why, but if we remove this line then sp.stats* functions won't work.